Zend/zend_call_stack.c

```diff
@@ -67,7 +67,9 @@ typedef int boolean_t;
 #define _STRUCTURED_PROC 1
```

Comment:
optional: if we want to be "nitpicky", sys/procfs.h is also not used then in case the libproc api is not used. Can be guarded as well alongside with the `#define _STRUCTURED_PROC 1` (which is for the sys/procfs.h to enable the newer procfs api). But again, it s being pedantic here :)

Reply:
Added in the next commit. Builds successfully.